apps/namadillo/src/App/Layout/SyncIndicator.tsx

```diff
@@ -66,14 +67,19 @@ export const SyncIndicator = (): JSX.Element => {
 
   return (
     <div className="relative group/tooltip px-1 py-3">
-      <div
-        className={twMerge(
-          "w-2 h-2 rounded-full",
-          "bg-green-500",
-          isSyncing && "bg-yellow-500 animate-pulse",
-          isError && !isSyncing && "bg-red-500"
-        )}
-      />
+      {isSyncing ?
```

Comment:
The pulse ring should be connected with the shielded sync only, not any sync (indexer, rpc, etc)